Library/Formula/git-ftp.rb

```diff
@@ -25,5 +23,17 @@ def install
       system "make", "-C", "man", "man"
       man1.install "man/man1/git-ftp.1"
     end
+    if build.with? "curl"
+      libexec.install bin/"git-ftp"
+      (bin/"git-ftp").write <<-EOS.undent
+        #!/bin/sh
+        PATH=#{Formula["curl"].opt_bin}:$PATH
+        #{libexec}/git-ftp "$@"
+      EOS
+    end
+  end
+
+  test do
+    system bin/"git-ftp --help"
```

Comment:
`system bin/"git-ftp", "--help"` would be preferred if you have a minute, thanks.